app/src/main/java/com/tomtom/sdk/examples/usecase/BasicNavigationActivity.kt

```diff
@@ -153,6 +154,7 @@ class BasicNavigationActivity : AppCompatActivity() {
             locationProvider = locationProvider,
             routeReplanner = routeReplanner
         )
+        (navigationConfiguration.vehicleProvider as DefaultVehicleProvider).setVehicle(Vehicle.Car())
```

Comment:
It can be set in the constructor of the `NavigationConfiguration`. It looks like by default it is already set to `Car`, so if your changes have some effect on behavior it looks like we have a bug.

Reply:
We just set it to Car in this example to show the developers where and how to set the vehicle type. In this case it does not have any effect since it was already Car before.